### PR TITLE
Updating the react demo 

### PR DESCRIPTION
**Pull request by sgsharma**
This updates the existing react demo:
- Adds a sign up form flow UI
- Updates the berbix SDK version

### CODE REVIEW

**Comment by benbeltzer** [comment]
@andrewsli @yding03 Sasha is working on updating some of our demo apps and has gotten started with this one! Could you please take a look at this and reach out to her if you have questions about it? Thanks!

File: client/src/App.tsx [hunk 1]
```diff
@@ -67,90 +95,52 @@ const App = () => {
...
+      </>
       )}
-      {error && <ErrorMessage>{error}</ErrorMessage>}
+    {error && <Typography>{error}</Typography>}
     </Container>
+    </Grid>
   );
 };
 
-const Container = styled.div`
```
Comment (andrewsli) [inline client/src/App.tsx:93]: 🎉

**Review by andrewsli — APPROVED** [review]
Thanks for getting rid of all those styled components! Assuming you've tested this locally, the changes look great.

**Review by yding03 — APPROVED** [review]
changes look good to me!

**Pull request merged**